### PR TITLE
Feature/mwb 164

### PR DESCRIPTION
**Pull request by duprijil**

### CODE REVIEW

File: tests/unit/backend/sparql_test_suite/test_sparql_validation.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        sparql_validation_adapter = SPARQLValidator(None)
+
+
+@pytest.mark.asyncio
+async def test_validate_test_data_with_sparql_test_suite(dummy_test_data_file_resource: TestDataFileResource,
+                                                         dummy_sparql_test_suite: SPARQLTestSuite):
+    result_test_data = await validate_test_data_with_sparql_test_suite(test_data=dummy_test_data_file_resource,
+                                                                       sparql_test_suite=dummy_sparql_test_suite)
+
+    assert result_test_data.sparql_validation_result is not None
```
Comment (CaptainOfHacks) [inline tests/unit/backend/sparql_test_suite/test_sparql_validation.py:27]: add test to check validation result content

File: tests/unit/backend/shacl_test_suite/test_shacl_validation.py [hunk 2]
```diff
@@ -34,7 +56,6 @@ async def test_validate_tests_data_with_shacl_tests(dummy_test_data_file_resourc
 def test_shacl_validator(dummy_test_data_file_resource: TestDataFileResource, dummy_shacl_test_suite: SHACLTestSuite):
     shacl_validator = SHACLValidator(test_data=dummy_test_data_file_resource)
 
-
     assert shacl_validator.validate(shacl_files=dummy_shacl_test_suite.file_resources) is not None
     assert shacl_validator.validate(None) is not None
```
Comment (CaptainOfHacks) [inline tests/unit/backend/shacl_test_suite/test_shacl_validation.py:60]: add tests to check validation results content

**Review by CaptainOfHacks — CHANGES_REQUESTED** [review]
Looks good, improve tests to cover validation results content, check concrete content.

File: tests/unit/backend/shacl_test_suite/test_shacl_validation.py (hunk 2 above)
Comment (duprijil) [reply tests/unit/backend/shacl_test_suite/test_shacl_validation.py]: done

File: tests/unit/backend/sparql_test_suite/test_sparql_validation.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+@pytest.mark.asyncio
+async def test_validate_tests_data_with_shacl_tests(dummy_test_data_file_resource: TestDataFileResource,
+                                                    dummy_sparql_test_suite: SPARQLTestSuite):
+    test_data_results = await validate_tests_data_with_shacl_tests(tests_data=[dummy_test_data_file_resource],
+                                                                  sparql_tests=dummy_sparql_test_suite.file_resources)
+
+    for result_test_data in test_data_results:
+        assert result_test_data.sparql_validation_result is not None
```
[the remark this answers is not in the document]
Comment (duprijil) [reply tests/unit/backend/sparql_test_suite/test_sparql_validation.py:50]: done

File: tests/unit/backend/sparql_test_suite/test_sparql_validation.py (hunk 1 above)
Comment (duprijil) [reply tests/unit/backend/sparql_test_suite/test_sparql_validation.py]: done

**Review by kaleanych — APPROVED** [review]

**Review by CaptainOfHacks — APPROVED** [review]

**Pull request merged**